app/controllers/assigned_applications_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
     )
   end
 
-  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength, Metrics/AbcSize
+  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength
```

Comment:
I think we could remove both of these opt outs:

1. Call it perhaps `next_application`
2. collapse some of these lines they aren't that long that it should matter

Reply:
👍